pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -78,8 +106,8 @@ let
   };
 
 # I think modelsim_ase/linux/vlm checksums itself, so use FHSUserEnv instead of `patchelf`
-in buildFHSUserEnv {
-  name = "quartus-prime-lite";
+in buildFHSUserEnv rec {
+  name = "quartus-wrapper";
```

Comment:
Maybe the directory where this nix file resides should change along this change? My opinion is not strong, but I tend to think it's not that important and it will introduce un-updated user installed derivations - because `nix-env` uses the `name` attribute to check for updates, not the attribute path (sigh).

Reply:
Oh. OK I will revert this then.